pages/dr9/sga.rst

```diff
@@ -10,178 +10,866 @@
 
 .. contents::
 
-*Page author and contact:* `John Moustakas`_
+*Primary contacts:* `John Moustakas (Siena College)`_ and `Dustin Lang (Perimeter Institute)`_.
 
-.. _`John Moustakas`: ../../contact/#other-experts
+.. _`John Moustakas (Siena College)`: ../../contact/#other-experts
+.. _`Dustin Lang (Perimeter Institute)`: ../../contact/#other-experts
 
-The Siena Galaxy Atlas (SGA) is an angular diameter-limited sample of galaxies constructed as part of the DESI Legacy Imaging Surveys. It provides
-custom, wide-area, optical and infrared mosaics (in :math:`grz` and :math:`W1-W4`), azimuthally averaged surface brightness profiles, and both aperture and integrated
-photometry for a sample of approximately 400,000 galaxies over :math:`20{,}000\,\mathrm{deg}^2`.
+`Data Access`_
+==============
 
-All the code used to generate and analyze the catalog is publicly available:
+The ``SGA-2020`` data products can be browsed and downloaded at the `SGA
+web-portal`_, both for individual galaxies and the full sample. Below, we
+describe the data model for these data products. A forthcoming paper (Moustakas,
+Lang, et al., in preparation) will describe the ``SGA-2020`` in detail.
+
+``Overview``
+============
+
+The ``Siena Galaxy Atlas (SGA)`` is a multiwavelength atlas of approximately
+385,000 nearby galaxies selected based on their apparent angular diameter.
+
+These galaxies are intrinsically large enough to be spatially resolved from our
+vantage point in the universe, providing a unique and powerful window into the
+fossil record of galaxy formation and evolution and the galaxy-halo
+connection. In addition, the atlas serves as a valuable angular map of the
+foreground extragalactic sky for cosmological surveys of distant galaxies like
+the `Dark Energy Spectroscopic Instrument (DESI)`_ survey.
+
+The ``SGA`` delivers precise coordinates, multiwavelength mosaics, azimuthally
+averaged optical surface brightness and color profiles, integrated and aperture
+photometry, model images and photometry, and additional metadata for the full
+sample. The primary sample relies on deep, wide-field *grz* imaging from the
```

Comment:
In some places you use `:math:grz` and in other places you use `*grz*`.

Reply:
There were instances of `:math:grz` in the original draft but none in the version you reviewed.